spark-3.1-spanner-lib/src/test/resources/db/insert_data.sql

```diff
@@ -56,3 +56,21 @@ VALUES
         "id2", [20, 200, 2991, 888885], ["A", "B", "C"], "this one", 93411, DATE("2023-09-23T00:00:00Z"),
         TIMESTAMP("2023-09-22T12:22:05Z"), false, [DATE("2023-09-02T00:00:00Z"), DATE("2023-12-31T00:00:00Z")],
         [TIMESTAMP("2023-09-22T12:11:10Z"), TIMESTAMP("2023-09-23T12:11:09Z")]);
+
+DELETE FROM bytesTable WHERE 1=1;
+INSERT INTO
+    bytesTable(id, A)
+VALUES
+   (1, B"ABCDEFGHIJKLMNOPQ"),
+   (2, B"abcdefghijklmnopq"),
+   (3, B"1234efghijklmnopq");
+
+
+DELETE FROM valueLimitsTable WHERE 1=1;
+INSERT INTO
+    valueLimitsTable(A, B, C, D, E)
+VALUES
+    (-9223372036854775808, CAST("NaN" AS FLOAT64), -9.9999999999999999999999999999999999999E+28, DATE("1700-01-01T00:00:00Z"), TIMESTAMP("9999-12-30T23:59:59.00Z")),
```

Comment:
Can we test the case 9999-12-31 23:59:59.999999999 ? and see what's the conversion to spark connector ?